### PR TITLE
PEP8 and PEP257 for Alphabet module

### PR DESCRIPTION
**Pull request by MarkusPiotrowski**
This pull request addresses issue #1473 D401: Use imperative mood... for the Alphabet module

### CODE REVIEW

File: Bio/Alphabet/Reduced.py [hunk 1]
```diff
@@ -6,23 +6,56 @@
...
 The Murphy tables are from here:
 
 Murphy L.R., Wallqvist A, Levy RM. (2000) Simplified amino acid
 alphabets for protein fold recognition and implications for folding.
 Protein Eng. 13(3):149-152
 
-Bio.utils.reduce_sequence is used to take a Protein alphabet, and reduce it using one of
-the tables here, or a user-defined table.
+These alphabets have been used with Bio.utils.reduce_sequence, which has been
+removed from Biopyhton. You can use this is alphabets and tables like this:
```
Comment (peterjc) [inline Bio/Alphabet/Reduced.py]: Typo (Biopyhton).

Comment (MarkusPiotrowski) [reply Bio/Alphabet/Reduced.py]: That's a classic, isn't it?

File: Bio/Alphabet/Reduced.py [hunk 2]
```diff
@@ -196,8 +258,14 @@ class HPModel(Alphabet.ProteinAlphabet):
...
+    Letters: A (Aliphatic: IVL), R (aRomatic: FYWH), C (Charged: KRDE),
+             T (Tiny: GACS), D (Diverse: TMQNP)
+    """
+
     letters = "ARCTD"
     size = 5
 
 
-hp_model = HPModel()
+pc5 = PC5()
```
Comment (peterjc) [inline Bio/Alphabet/Reduced.py]: That does look like a bug, doesn't it.

**Comment by peterjc** [comment]
Looks good - can you add yourself to the ``NEWS.rst`` for the next release please?

File: Bio/Alphabet/Reduced.py (hunk 2 above)
Comment (MarkusPiotrowski) [reply Bio/Alphabet/Reduced.py]: Yes, that was a bug, ```hp_model = HPModel()``` is already called after defining the ```HPModel``` class.

**Comment by MarkusPiotrowski** [comment]
I'm already in the ```NEWS.rst``` file for release 1.71.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**